api-tests/data/common/headers.js

```diff
@@ -1,4 +0,0 @@
-module.exports.jsonContent = {
-    'Content-Type': 'application/json',
-    'X-Version': 1.0,
-}
```

Comment:
Hi @mdp0 not sure why this file was deleted?? I have added this in order to get the header components in the request to be read by the other services.

I might need a walkthrough of the PR as many changes have been done on the api tests that qa have written.

Reply:
Removed as it wasn't in use anywhere.

Reply:
I was using it to export the json header but right now exporting the modules seems to be doing the job :)